### ISSUE TITLE
Module name conflict: 'Force' from gampleman/elm-visualization and ianmackenzie/elm-units

### ISSUE THREAD

**Issue by ianmackenzie** [elm/compiler#1887]
As an additional example for #1625, there has been another potential module name conflict found between `gampleman/elm-visualization` (which has a [`Force`](https://package.elm-lang.org/packages/gampleman/elm-visualization/latest/Force) module for doing force-based particle simulation) and `ianmackenzie/elm-units` (which has a [`Force`](https://package.elm-lang.org/packages/ianmackenzie/elm-units/latest/Force) module for constructing and converting between force values in different units).

This has not yet led to an actual reported conflict in user code, but it would seem quite natural to want to  use both packages in a scientific/technical visualization app, and at least @dmy [has expressed interest in using both packages together](https://discourse.elm-lang.org/t/namespace-company-modules/2965/4).

**Comment by rlefevre** [comment]
Meta issue here: https://github.com/elm/compiler/issues/1625.

**Comment by rlefevre** [comment]
@evancz I wanted to add this example to https://github.com/elm/compiler/issues/1625 but I realized it would likely change your conclusion about the best case. Still it would be useful to have it there (more over references to this bug do not appear anymore, maybe because of closed comments).

 - `Force` from `ianmackenzie/elm-units` and `gampleman/elm-visualization` in https://github.com/elm/compiler/issues/1887

**Comment by 13tales** [comment]
I've been bitten by this; I want to use the `Force` module from the `elm-visualization` package to add a force-directed graph to my project, but can't, because I'm already using `elm-units` which exposes a module with the same name 😞

**Comment by 13tales** [comment]
@ianmackenzie Oh, just noticed that you've published `elm-units-prefixed` as a workaround. Thanks! 🙏

**Comment by ianmackenzie** [comment]
Yes, and I now use `elm-units-prefixed` at work because we have both `elm-units` and `elm-visualization` in our codebase. It's worked pretty well, although I needed to add `Quantity.unsafe` and `Quantity.unwrap` functions since it's not possible to alias constructors/non-opaque types in the same way that you can alias functions, values and opaque types. (If `elm-units` contained an exposed custom type with more than one constructor, then wrapping it would have been much trickier.)
